pkg/cortex/modules.go

```diff
@@ -170,6 +171,23 @@ func (t *Cortex) initOverrides() (serv services.Service, err error) {
 	return nil, err
 }
 
+func (t *Cortex) initOverridesExporter() (services.Service, error) {
+	cfgChan := tenantLimitsRuntimeConfigChannel(t.RuntimeConfig)
+	if cfgChan == nil {
+		// This target isn't enabled by default ("all") and requires runtime configuration
+		// to work. Fail if it can't be setup correctly since the user explicitly wanted this
+		// target to run.
+		return nil, errors.New("runtime configuration not available for overrides exporter")
```

Comment:
I'd suggest to clarify error message: "overrides-exporter has been enabled, but no runtime configuration file was configured"